src/main/cheffy/account.clj

```diff
@@ -0,0 +1,25 @@
+(ns cheffy.account
+  (:require
+   [cheffy.interceptors :as interceptors]
+   [io.pedestal.interceptor.chain :as chain]
+   [io.pedestal.interceptor.helpers :refer [before]]
+   [ring.util.response :as response]))
+
+(defn on-request [{:keys [request] :as ctx}]
+  (let [{:keys [system/auth transit-params]} request
+        {:keys [email password]} transit-params
+        account-result (auth/create-cognito-account auth
+                                                    {:email email
+                                                     :password password})]
+    (if (contains? account-result :cognitect.anomalies/category)
+      (response/bad-request {:type (:__type account-result)
+                             :message (:message account-result)
+                             :data {:account-id email}})
```

Comment:
In 58. lesson they "refactor" this to throw an exception but I'm not doing that for now